packages/rocketchat-e2e/package.js

```diff
@@ -18,6 +18,8 @@ Package.onUse(function(api) {
 	]);
 
 	api.mainModule('client/rocketchat.e2e.js', 'client');
+	api.addFiles('client/events.js', 'client');
```

Comment:
Can you move these imports (addFiles) to inside the main module of the respective side (client/server)?